packages/peregrine/lib/talons/ProductListing/useProductListing.js

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from '@apollo/react-hooks';
+
+import { useCartContext } from '../../context/cart';
+
+export const useProductListing = props => {
+    const { query } = props;
+
+    const [{ cartId }] = useCartContext();
+
+    const { data, error, loading } = useQuery(query, {
+        variables: { cartId },
+        // temporarily use fetch policy that validates against the network
+        // until all cart mutations are correctly updating the cache
+        fetchPolicy: 'cache-and-network'
```

Comment:
I think I'd rather just remove the fetchPolicy that made development easier, because I don't know the big picture of re-factoring all these other cart components to start monitoring Apollo cache instead of global state.

Reply:
Removed in 0ac27f8